src/Header/Nav.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import '../Styles/Nav.scss';
+import logo from './logo.png'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+
+export default function Nav(props) {
```

Comment:
Are any props being passed to Nav? Like the semantic HTML - good use of it. We could also the `title` attributes to the icons that link to social media (accessibility).